pkgs/by-name/u-/u-config/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "u-config";
+  version = "0.33.1";
+
+  src = fetchFromGitHub {
+    owner = "skeeto";
+    repo = "u-config";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-r1zcXKLqw/gK+9k3SX7OCBaZhvV2ya5VC9O3h+WdkyY=";
+  };
+
+  makeFlags = [
+    "CROSS=${stdenv.cc.targetPrefix}"
+    "CC=${lib.getExe stdenv.cc}"
+  ];
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildFlags = [ "pkg-config"];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 pkg-config -t $out/bin
```

Comment:
Why isn't this done automatically?

Reply:
I dont understand your question

Reply:
Normally, this would be copied to `$out/bin` automatically by some hook. Does this just use some sort of weird build system?

Reply:
The install rule of this project would cause it to attempt installing window cross-compiled binary. Installation for linux wasn't declared

Reply:
Makes sense. Feel free to resolve this.